### PR TITLE
Release 0.44.0

### PR DESCRIPTION
**Pull request by lfittl**

### CODE REVIEW

**Comment by uhoh-itsmaciek** [comment]
@lfittl should we try to get #273 in here as well?

**Comment by lfittl** [comment]
@uhoh-itsmaciek Yep!

I think we should get all currently open PRs (#273, #284, #285, #286) included in this collector release. Hoping to wrap those PRs up tomorrow.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+    and their parameters marked as `<removed>`.
+  - This setting is recommended when EXPLAIN plans may contain sensitive
+    data that should not be stored. Please verify that the logic works
+    as expected with your workload and log output.
+* Be more accepting with outdated pg_stat_statements versions
+  - With this change, its no longer required to run
+    "ALTER EXTENSION pg_stat_statements UPDATE" in order to use
+    the collector after a Postgres upgrade
+  - The collector will output an info message in case an outdated
+    pg_stat_statements schema is in use
```
Comment (uhoh-itsmaciek) [inline CHANGELOG.md:21]: If an outdated *version* is in use, no? Separately, I think we also handle the extension installed in a different schema?

Comment (lfittl) [reply CHANGELOG.md:21]: Yep, good point.

**Review by uhoh-itsmaciek — APPROVED** [review]

**Pull request merged**